### DISCUSSION TITLE
Trying to create a Rocky Linux VM

### DISCUSSION THREAD

**Discussion by SimplySeth** [cirruslabs/tart#706] [General]
I tried this command: 

`tart run --disk ./Rocky-8.9-aarch64-minimal.iso rockylinux8`

screen comes up I choose Install and it just doesn't boot, just goes back to the boot options
Anything obvious I'm doing wrong?

**Comment by fkorotkov** [comment]
Have you tried Rocky Linux 9? It seems 8 might [not play well](https://forums.rockylinux.org/t/rocky-linux-arm64-not-booting/2433) with the `Virtualization.Framework`.

**Reply by SimplySeth** [reply]
Thank you. I got the OS installed with Rocky 9
